### DISCUSSION TITLE
Is there a way to get a contact form on Pages?

### DISCUSSION THREAD

**Discussion by THEJON78** [community/community#66067] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I just want to be able to do a contact form in Pages. I don't know much html.

**Comment by AnthonyCarafa** [comment]
I believe so, but I forget how.

**Comment by nirav-gajera** [comment, marked as answer]
ref: https://www.123formbuilder.com/html-contact-form/#:~:text=Creating%20a%20Working%20HTML%20Contact%20Form%201%201.,3%203.%20Add%20the%20PHP%20contact%20form%20script

https://www.geeksforgeeks.org/design-a-contact-us-page-using-html-and-css/

**Comment by THEJON78** [comment]
> ref: https://www.123formbuilder.com/html-contact-form/#:~:text=Creating%20a%20Working%20HTML%20Contact%20Form%201%201.,3%203.%20Add%20the%20PHP%20contact%20form%20script
> 
> https://www.geeksforgeeks.org/design-a-contact-us-page-using-html-and-css/

Thanks!
